src/canvas-force-graph.js

```diff
@@ -49,6 +50,7 @@ export default Kapsule({
     linkWidth: { default: 1, triggerUpdate: false },
     linkCurvature: { default: 0, triggerUpdate: false },
     linkCanvasObject: { triggerUpdate: false },
+    linkCanvasObjectMode: { triggerUpdate: false },
```

Comment:
This one is missing the default val.

Reply:
Yep, I didn't do any of the changes proposed for links. I was waiting to see a way to do the rendering properly.